src/mcts/stoppers/factory.cc

```diff
@@ -55,17 +55,30 @@ const OptionId kSlowMoverId{
     "Budgeted time for a move is multiplied by this value, causing the engine "
     "to spend more time (if value is greater than 1) or less time (if the "
     "value is less than 1)."};
-const OptionId kTimeMidpointMoveId{
-    "time-midpoint-move", "TimeMidpointMove",
-    "The move where the time budgeting algorithm guesses half of all "
-    "games to be completed by. Half of the time allocated for the first move "
-    "is allocated at approximately this move."};
-const OptionId kTimeSteepnessId{
-    "time-steepness", "TimeSteepness",
-    "\"Steepness\" of the function the time budgeting algorithm uses to "
-    "consider when games are completed. Lower values leave more time for "
-    "the endgame, higher values use more time for each move before the "
-    "midpoint."};
+const OptionId kTimeFactorId{"time-factor", "TimeFactor",
```

Comment:
`initial-slowmover`?
At least the help needs to be changed. s/start allocations/first moves/?